### ISSUE TITLE
Torus courses should have consistent screen reader support

### ISSUE THREAD

**Issue by eliknebel** [Simon-Initiative/oli-torus#2887]
Math course authors was a uniform consistency for mathematics screen reader. We should ensure that course content in torus is accessible to all screen readers, specifically LaTex support. Investigate the different types of screen readers that exist and produce a compatibility report.
